lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -255,6 +266,16 @@ namespace :foreman_tasks do
         tasks.each_with_index do |task, count|
```

Comment:
How about we collect the tasks which were successfully exported here and then pass those to the function which builds the index?

Reply:
That's a great idea. I'll work on this now. Thanks!

Reply:
Done. I added an exported_tasks array where I pushed each successfully exported task into, and I'm now passing generate_index() the new exported_tasks array instead of the original tasks array (or array-like structure).

Looking forward to your review. Thanks again for this, @adamruzicka !